config/config.go

```diff
@@ -7,19 +7,21 @@ import (
 )
 
 const (
-	LOG_DATE_FORMAT_KEY          = "FLOGO_LOG_DTFORMAT"
-	LOG_DATE_FORMAT_DEFAULT      = "2006-01-02 15:04:05.000"
-	LOG_LEVEL_KEY                = "FLOGO_LOG_LEVEL"
-	LOG_LEVEL_DEFAULT            = "INFO"
-	RUNNER_TYPE_KEY              = "FLOGO_RUNNER_TYPE"
-	RUNNER_TYPE_DEFAULT          = "POOLED"
-	RUNNER_WORKERS_KEY           = "FLOGO_RUNNER_WORKERS"
-	RUNNER_WORKERS_DEFAULT       = 5
-	RUNNER_QUEUE_SIZE_KEY        = "FLOGO_RUNNER_QUEUE"
-	RUNNER_QUEUE_SIZE_DEFAULT    = 50
-	APP_CONFIG_LOCATION_KEY      = "FLOGO_CONFIG_PATH"
-	APP_CONFIG_LOCATION_DEFAULT  = "flogo.json"
-	STOP_ENGINE_ON_ERROR_KEY     = "STOP_ENGINE_ON_ERROR"
+	LOG_DATE_FORMAT_KEY               = "FLOGO_LOG_DTFORMAT"
+	LOG_DATE_FORMAT_DEFAULT           = "2006-01-02 15:04:05.000"
+	LOG_LEVEL_KEY                     = "FLOGO_LOG_LEVEL"
+	LOG_LEVEL_DEFAULT                 = "INFO"
+	RUNNER_TYPE_KEY                   = "FLOGO_RUNNER_TYPE"
+	RUNNER_TYPE_DEFAULT               = "POOLED"
+	RUNNER_WORKERS_KEY                = "FLOGO_RUNNER_WORKERS"
+	RUNNER_WORKERS_DEFAULT            = 5
+	RUNNER_QUEUE_SIZE_KEY             = "FLOGO_RUNNER_QUEUE"
+	RUNNER_QUEUE_SIZE_DEFAULT         = 50
+	APP_CONFIG_LOCATION_KEY           = "FLOGO_CONFIG_PATH"
+	APP_CONFIG_LOCATION_DEFAULT       = "flogo.json"
+	STOP_ENGINE_ON_ERROR_KEY          = "STOP_ENGINE_ON_ERROR"
+	PROPERTY_DELIMITER_FORMAT_KEY     = "FLOGO_PROPERTY_DELIMITER"
+	PROPERTY_DELIMITER_FORMAT_DEFAULT = "{*}"
```

Comment:
The latest proposal for resolution on environment variables is ${env.VARIABLE_NAME}.

So the delimiter should be ${*}